Products/CMFPlone/browser/templates/plone-overview.pt

```diff
@@ -31,14 +31,9 @@
                 i18n:attributes="alt" /></p>
         <h1 i18n:translate="">Plone is up and running.</h1>
         <p class="lead">
-            <span i18n:translate="label_plone_com_description"> For an introduction to Plone, success stories, demos, providers, visit</span>
-            <a href="http://plone.com"
-              title="Plone.com"
-              target="_blank"
-              i18n:attributes="title label_plone_com_title;">plone.com</a>.
-            <span i18n:translate="label_plone_org_description"> For documentation, add-ons, support, community, visit</span>
+            <span i18n:translate="label_plone_org_description"> For an introduction to Plone, documentation, demos, add-ons, support, and community, visit</span>
             <a href="http://plone.org"
-              title="Plone Community Home"
+              title="Plone CMS: Open Source Content Management"
```

Comment:
Please revert this change, per https://github.com/plone/Products.CMFPlone/pull/3607#discussion_r945289380

Reply:
changed!